python-sdk/example_dags/calculate_popular_movies_by_genre.py

```diff
@@ -0,0 +1,36 @@
+import tempfile
+from datetime import datetime
+
+import pandas as pd
+from airflow import DAG
+
+from astro import sql as aql
+from astro.files import File
+from astro.sql.operators.export_to_file import ExportToFileOperator
+from astro.table import Table
+
+
+@aql.dataframe(columns_names_capitalization="original")
+def top_movies_by_genre(input_df: pd.DataFrame):
+    top_movies = []
+    for _, genre_df in input_df.groupby("genre1"):
+        genre_df.sort_values(by="rating", ascending=False)[["title", "rating", "genre1"]].head(5)
+        top_movies.append(genre_df)
+    return top_movies
+
+
+with DAG(
+    "calculate_popular_movies_by_genre",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    catchup=False,
+) as dag:
+    imdb_movies = aql.load_file(
+        File("https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb_v2.csv"),
```

Comment:
We can use S3 URL instead, as HTTP URL sometimes makes the tests flaky.

Reply:
@sunank200, could you point out when the HTTP tests were flaky? This is the first time I have heard about this! HTTP tests are useful because users can run these DAGs without additional providers or permissions.